src/server.ts

```diff
@@ -1,11 +1,12 @@
 import App from '@/app';
 import AuthRoute from '@routes/auth.route';
+import GroupsRoute from './routes/group.route';
```

Comment:
Better to follow a common pattern of route imports and import it as following:
```suggestion
import GroupsRoute from '@routes/group.route';
```